ontopy/ontology.py

```diff
@@ -1010,9 +1026,9 @@ def save(
                     graph.remove((s, p, o))
                     graph.add((URIRef(self.iri), p, o))
 
-            graph.serialize(destination=filename, format=format)
+            graph.serialize(destination=filepath, format=format)
```

Comment:
Seems that we should add a test that saves a squashed ontology...

Reply:
You made that at the beginning of this week....

Reply:
Codecov just marked that squashing was not tested. But that is gone now, so the test seems to have been readded now.